components/molecule/thumbnail/src/index.js

```diff
@@ -0,0 +1,118 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import ImagePlaceholder from '@schibstedspain/sui-image-placeholder'
+import cx from 'classnames'
+
+const BASE_CLASS = 'sui-MoleculeThumbnail'
+const CAPTION_CLASS = `${BASE_CLASS}-caption`
+
+const SIZES = {
+  LARGE: 'large',
+  MEDIUM: 'medium',
+  SMALL: 'small',
+  XSMALL: 'xsmall'
+}
+
+const RATIOS = {
+  '1:1': '1-1',
+  '4:3': '4-3',
+  '16:9': '16-9'
+}
+
+const SHAPES = {
+  SQUARED: 'squared',
+  CIRCLED: 'circled'
+}
+
+class MoleculeThumbnail extends Component {
+  _renderFigure () {
+    return (
+      <figure className={
+        cx(
+          `${BASE_CLASS}`,
+          `${BASE_CLASS}--${this.props.size}`,
+          `${BASE_CLASS}--${this.props.ratio}`,
+          `${BASE_CLASS}--${this.props.shape}`
+        )}>
+        <ImagePlaceholder
+          {...this.props}
+        />
+        {
+          this.props.captionText &&
+            <figcaption className={CAPTION_CLASS}>
+              {this.props.captionText}
+            </figcaption>
+        }
+      </figure>
+    )
+  }
+  render () {
+    return this.props.href
+      ? (
+        <a href={this.props.href} target={this.props.target}>
```

Comment:
Please use a [LinkFactory](https://github.com/SUI-Components/sui-components/blob/bd8cd89d9fb67559e627c35a5b3a9adeff804896/components/atom/button/src/index.js#L148) instead of an anchor, we might want to use react router links instead of this

also please add rel noopener like [here](https://github.com/SUI-Components/sui-components/blob/bd8cd89d9fb67559e627c35a5b3a9adeff804896/components/atom/button/src/index.js#L62) in case target = _blank